bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/info-table.html.twig

```diff
@@ -0,0 +1,77 @@
+{# @var $element \Pimcore\Model\Element\AbstractElement #}
+{% set language = element.getProperty('language') %}
+<div class="data-table <?= $this->cls ?>">
+    <table>
+        {% if element is instanceof ('\\Pimcore\\Model\\DataObject\\Concrete') %}
+            <tr>
+                <th>{{ 'class'|trans([],'admin') }}</th>
+                <td>{{ element.getClassName() }} [{{ element.getClassId() }}]</td>
+            </tr>
+        {% endif %}
+
+        {% if element is instanceof ('\\Pimcore\\Model\\Asset') %}
+            <tr>
+                <th>{{ 'mimetype'|trans([],'admin') }}</th>
+                <td>{{ element.getMimetype() }}</td>
+            </tr>
+        {% endif %}
+
+        {% if language is not empty %}
+            <tr>
+                <th>{{ 'language'|trans([],'admin') }}</th>
+                <td style="padding-left: 40px; background: url({{ pimcore_language_flag(language, false) }}) left top no-repeat; background-size: 31px 21px;">
+                    {% set locales = callStatic('\\Pimcore\\Tool', 'getSupportedLocales') %}
```

Comment:
`callStatic` is not good DX, better create twig extensions:

`pimcore_supported_locales`